### DISCUSSION TITLE
Enabling mandatory two-factor authentication (2FA) requirement for z/OS Open Tools on June 1, 2023

### DISCUSSION THREAD

**Discussion by IgorTodorovskiIBM** [zopencommunity/meta#304] [Announcements]
Hi,

Starting **June 1, 2023**, we are implementing a mandatory two-factor authentication (2FA) requirement for all of our [z/OS Open Tools organization](https://github.com/ZOSOpenTools).

### Why?
To reduce the risk of unauthorized access and prevent malicious activities that could compromise our organization's repositories and settings.

### Action Required:
[Enable 2FA](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa) for your personal [GitHub.com](http://github.com/) account before June 1, 2023.

### Important Points:
* Failure to enable 2FA by the deadline will result in automatic removal from the organization.
* If removed, you can reinstate access within three months by enabling 2FA.
* Disabling 2FA after the requirement is enabled will lead to automatic removal.
* Note: Starting in March 2023 and through the end of 2023, GitHub will gradually begin to require all users who contribute code on [GitHub.com](http://github.com/) to enable one or more forms of two-factor authentication (2FA).

For more information and instructions, refer to GitHub's "[Securing your account with two-factor authentication (2FA)](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa)" documentation.

We appreciate your cooperation in maintaining a secure environment!

**Comment by MikeFultonDev** [comment]
Since it is after June 1st, should we add this to our docs if not there already so people know when they start to do development @IgorTodorovskiIBM ?
